common/v2/utils/generateUUID.tsx

```diff
@@ -24,6 +24,9 @@ export const generateAssetUUID = (chainId: string | number, address?: string): T
     ? (getUuid(`${chainId.toString()}-${toChecksumAddress(address)}`) as TUuid)
     : (getUuid(chainId.toString()) as TUuid);
 
-export const generateContractUUID = (abi: string) => getUuid(abi) as TUuid;
+export const generateContractUUID = (id: NetworkId, address: string, abi: string) =>
+  address
+    ? (getUuid(`${id}-${toChecksumAddress(address)}-${abi}`) as TUuid)
+    : (getUuid(`${id}-${abi}`) as TUuid);
```

Comment:
We should remove the abi part of this to stick with the uuid convention we follow with assets. We should do the same with user accounts if we aren't already doing this, as well.

Reply:
I removed a network object from each account. The settings-export filesize decreased a lot :) Nice catch!

For generating UUIDs I removed abi from contracts generating, created a similar method for accounts and apply those methods (for assets/account/contract) throughout the project.